examples/demo_runner.py

```diff
@@ -338,6 +339,14 @@ def init_common(self):
         random.seed(self._sim_settings["seed"])
         self._sim.seed(self._sim_settings["seed"])
 
+        recompute_navmesh = self._sim_settings.get("recompute_navmesh")
+        if recompute_navmesh or not self._sim.pathfinder.is_loaded:
+            if not self._sim_settings["silent"]:
+                print("Recomputing navmesh")
+            navmesh_settings = habitat_sim.NavMeshSettings()
+            navmesh_settings.set_defaults()
```

Comment:
This will not adjust to agent params, but we don't include those in settings anyway.

Reply:
Exactly my thoughts.